website/templates/profile/notifications.mako

```diff
@@ -65,7 +74,9 @@
     <% import website %>
     ${parent.javascript()}
     <script type="text/javascript">
-        window.contextVars = $.extend({}, window.contextVars, {'mailingList': ${website.settings.MAILCHIMP_GENERAL_LIST | sjson, n }});
+        window.contextVars = $.extend({}, window.contextVars, {
+            'mailingLists': [${website.settings.MAILCHIMP_GENERAL_LIST | sjson, n }, ${website.settings.OSF_HELP_LIST | sjson, n }]
```

Comment:
It is arguably safer to json-encode a python list rather than json-encode single values within `[]`.

I think the following should work, but please verify.

```
mailingLists: ${ [website.settings.MAIL..., website....] | sjson, n}
```